source/qx/io/jsonrpc/__init__.js

```diff
@@ -1,116 +0,0 @@
-/**
- * This namespace provides a JSON Remote Procedure Call (JSON-RPC) version 2 API .
- * See https://www.jsonrpc.org/specification
- *
- * JSON-RPC v2 is transport-agnostic. We provide an API interface
- * and an HTTP transport implementatio. Other transports
- * based on websockets or other mechanisms can be added later.
- *
- * There are two ways of working with this API.
- *
- * 1) Using the transport objects
- *
- * You can either work with the transports directly. For example, you can send
- * out requests, notifications, and batches like so:
- *
- * <pre>
- *   (async()=>{
- *    const transport = new qx.io.jsonrpc.transport.Http("https://domain.com/endpoint");
- *    transport.getRequest().setAuthentication(new qx.io.request.authentication.Bearer("TOKEN"));
- *    transport.getRequest().setTimeout(10000);
- *    const request = new qx.io.jsonrpc.message.Request("some_method", ["first-param","second-param"]);
- *    const notification = new qx.io.jsonrpc.message.Notification("other_method", [1,2,3]);
- *    let result;
- *    try {
- *      transport.send(request);
- *      transport.send(notifiation);
- *      result = await request.getPromise();
- *    } catch(e) {
- *      // handle exceptions
- *    }
- *   })();
- * </pre>
- *
- * or using a batch, like so:
- *
- * <pre>
- *   (async()=>{
- *    const transport = new qx.io.jsonrpc.transport.Http("https://domain.com/endpoint");
- *    const batch = ()new qx.io.jsonrpc.message.Batch())
- *      .addRequest("method3")
- *      .addNotification("method4")
- *      .addRequest("method5","foo)
- *      .addRequest("method6", [{foo:"bar"}]);
- *    let results;
- *    try {
- *      transport.send(batch);
- *      results = await qx.Promise.all(batch.getPromises());
- *      // results will be an array with three items, the result of the requests
- *    } catch(e) {
- *      // handle exceptions
- *    }
- *   })();
- * </pre>
- *
- * 2) Using qx.io.jsonrpc.Client
- *
- * qx.io.jsonrpc.Client is an API built on top of the transports. It
- * will figure out the type of transport from the URI passed to it,
- * and provides a way to prepend a "service name" to all the method
```

Comment:
The `service name` was a construct of qx1. It's not part of version 2.0, so probably doesn't belong in here at all.